addon/components/sl-calendar-month.js

```diff
@@ -81,11 +73,14 @@ export default Ember.Component.extend({
     /**
      * The short string name of the represented month
      *
-     * @function shortName
-     * @returns  {Ember.String}
+     * @function
+     * @observes month
```

Comment:
If it's planned for the future, why would we remove the tag usage at this point?